src/App.tsx

```diff
@@ -17,25 +18,37 @@ type PagesType = {
     [key in PageName]: React.ReactNode;
 };
 
-const LS_KEY = 'cartMasterAcademy';
-
-const localStorageCart = localStorage.getItem(LS_KEY) ? JSON.parse(localStorage.getItem(LS_KEY)!) : [];
+const LS_KEY_CART = 'MasterAcademyCart';
+const LS_KEY_THEME_MODE = 'MasterAcademyThemeMode';
 
 const App: FC = () => {
     const [pageActive, setPageActive] = useState<PageName>(PageName.ABOUT);
-    const [productsInCart, setProductsInCart] = useState<Product[]>(localStorageCart);
 
-    useEffect(() => {
-        const lsCart = localStorage.getItem(LS_KEY);
-        if (lsCart) {
-            setProductsInCart(JSON.parse(lsCart));
+    const [currentTheme, setCurrentTheme] = useState<ThemeMode>(() => {
+        const isUserSystemThemeLight = window.matchMedia('(prefers-color-scheme: light)').matches;
+        const lsThemeMode = localStorage.getItem(LS_KEY_THEME_MODE) as ThemeMode;
+
+        if (lsThemeMode) {
+            return JSON.parse(lsThemeMode);
+        } else {
+            return isUserSystemThemeLight ? ThemeMode.LIGHT : ThemeMode.DARK;
```

Comment:
You can make a `theme.service.ts` or `theme.helper.ts` file and move to it all logic related to the theme.
E.g. this could be `getSystemTheme`